care/emr/resources/questionnaire/utils.py

```diff
@@ -15,6 +15,9 @@
 from care.emr.resources.observation.spec import ObservationSpec, ObservationStatus
 from care.emr.resources.questionnaire.spec import QuestionType
 
+BOOLEAN_TRUE_STRINGS = ("true", "on", "ok", "y", "yes", "1")
```

Comment:
Hmm, enable when UI is being reworked on FE side, so for boolean, user will have a dropdown (w/ true and false as values) 👍 

Would be great if you could great a migration file to migrate existing yes/no etc to true/false :)

Should only be yes or no iirc, but might want to connect with Vysakh and check.